fxa-auth-db-server/test/backend/remote.js

```diff
@@ -612,6 +612,20 @@ module.exports = function(cfg, server) {
           var devices = r.obj
           t.equal(devices.length, 1, 'devices contains one item again')
 
+          return client.postThen('/account/' + user.accountId + '/device/' + user.deviceId + '/update', {
+            name: '4a6f686e'
+          })
+        })
+        .then(function(r) {
+          respOk(t, r)
+          return client.getThen('/account/' + user.accountId + '/devices')
+        })
+        .then(function(r) {
+          respOk(t, r)
+          var devices = r.obj
+          t.equal(devices.length, 1, 'devices contains one item again')
+          t.equal(devices[0].name, '4a6f686e', 'name was not automagically bufferized')
```

Comment:
This is a new test I added, and which failed against the previous version of the code.